tasks/scripting/src/main/java/org/infinispan/scripting/impl/ScriptingManagerImpl.java

```diff
@@ -150,9 +150,12 @@ boolean containsScript(String taskName) {
          if (authorizationManager != null) {
             authorizationManager.checkPermission(AuthorizationPermission.EXEC, metadata.role().orElse(null));
          } else {
-            authorizer.checkPermission(AuthorizationPermission.EXEC, metadata.role().orElse(null));
+            if (context.getSubject().isPresent()) {
```

Comment:
this is the wrong fix. The problem is in the Rest enpoint.

Reply:
but I'm not using REST but hotrod @pruivo

Reply:
just ignore me :rofl: